clients/src/test/java/org/apache/kafka/clients/NetworkClientTest.java

```diff
@@ -45,19 +49,42 @@
 
 public class NetworkClientTest {
 
-    private final int requestTimeoutMs = 1000;
-    private MockTime time = new MockTime();
-    private MockSelector selector = new MockSelector(time);
-    private Metadata metadata = new Metadata(0, Long.MAX_VALUE);
-    private int nodeId = 1;
-    private Cluster cluster = TestUtils.singletonCluster("test", nodeId);
-    private Node node = cluster.nodes().get(0);
-    private long reconnectBackoffMsTest = 10 * 1000;
-    private NetworkClient client = new NetworkClient(selector, metadata, "mock", Integer.MAX_VALUE, reconnectBackoffMsTest, 
-            64 * 1024, 64 * 1024, requestTimeoutMs, time);
-    
-    private NetworkClient clientWithStaticNodes = new NetworkClient(selector, new ManualMetadataUpdater(Arrays.asList(node)),
-            "mock-static", Integer.MAX_VALUE, 0, 64 * 1024, 64 * 1024, requestTimeoutMs, time);
+    protected final int requestTimeoutMs = 1000;
+    protected MockTime time = new MockTime();
+    protected MockSelector selector = new MockSelector(time);
+    protected Metadata metadata = new Metadata(0, Long.MAX_VALUE);
+    protected int nodeId = 1;
+    protected Cluster cluster = TestUtils.singletonCluster("test", nodeId);
+    protected Node node = cluster.nodes().get(0);
+    protected long reconnectBackoffMsTest = 10 * 1000;
+    protected NetworkClient client = getNetworkClient();
+
+    private NetworkClient clientWithStaticNodes = getNetworkClientWithStaticNodes();
+
+    private NetworkClient getNetworkClient() {
```

Comment:
Nit: maybe this should be `createNetworkClient` and similar for the other method.